backend/src/packages/trucks/truck.controller.ts

```diff
@@ -140,7 +140,7 @@ class TruckController extends Controller {
       path: TruckApiPath.ROOT,
       method: 'POST',
       validation: {
-        body: truckCreateRequestBody,
+        body: { ...truckCreateRequestBody },
```

Comment:
```suggestion
        body: truckCreateRequestBody,
```
why?

Reply:
It's a typo, added businessId, and forgot to remove destructuring